### PR TITLE
Quering string array example added #issue 221

### PR DESCRIPTION
**Pull request by Ramin-Guliyev**

### CODE REVIEW

File: examples/Redis.OM.QueryingStringsWithinArrays/Program.cs [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+//Contains usage 
+var filteredExperiences = await experiences.Where(x=>x.Skills.Contains("C#")).ToListAsync();
+
+foreach (var item in filteredExperiences)
+{
+    Console.WriteLine(item.ToString());
+}
+
+Console.ReadKey();
```
Comment (rtynski) [inline examples/Redis.OM.QueryingStringsWithinArrays/Program.cs:18]: Add new line

**Comment by rtynski** [comment]
Proposal:
- Directory Helpers i Models (as in Redis.OM.BasicMatchingQueries)

File: examples/Redis.OM.QueryingStringsWithinArrays/Experience.cs [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    [RedisIdField]
+    [Indexed]
+    public string Id { get; set; } = default!;
+
+    [Indexed]
+    public string[] Skills { get; set; } = default!;
+
+    public override string ToString()
+    {
+        return string.Join(" ", Skills.Skip(0));
```
Comment (slorello89) [inline examples/Redis.OM.QueryingStringsWithinArrays/Experience.cs:17]: calling `skip` with 0 is inert I think - so I'd say remove the Skip here.

File: examples/Redis.OM.QueryingStringsWithinArrays/Program.cs (hunk 1 above)
Comment (slorello89) [inline examples/Redis.OM.QueryingStringsWithinArrays/Program.cs:18]: This is just personal preference, but I'd say remove the ReadKey at the end here and just let the program terminate when it's done.

**Review by slorello89 — CHANGES_REQUESTED** [review]
Looking pretty good @Ramin-Guliyev - just a couple of minor items and then I think we're good to go!

**Comment by Ramin-Guliyev** [comment]
@slorello89 I have done it. Please review again and merge :)

**Review by slorello89 — APPROVED** [review]
LGTM 👍 thanks for your contribution @Ramin-Guliyev happy 🎃 hacktoberfest 🎃 (will merge when CI is done)

**Comment by SuzeShardlow** [comment]
@Ramin-Guliyev thanks for your PR!  If you are a member of our Discord server (https://discord.gg/redis), please let us know your handle so we can give you the `Hacktoberfest 2022` role :D

**Pull request merged**